server/src/config/config.ts

```diff
@@ -1,6 +1,26 @@
 const config = {
     app: {
-        port: 4000
+        port: 4000,
```

Comment:
think we need to make the config the same principle as in `client/src/config`: Default to remote, provide local values commented out. And source the default ones from ENV first, then fallback value.

Or maybe it makes sense to have a global config sourced by client and server config. But think won't work on client sourcing from files from outside of `src/` because of a CRA restriction

Reply:
Resolved in 5c4f617 and c6ead43

Reply:
🚀